web/js/layoutManager.js

```diff
@@ -50,11 +50,12 @@ LayoutViewport, ItemsHandler */
     return isScreen ? HANGOUT_BY_DEFAULT : userSelectedLayout;
   }
 
-  function getLayoutByScreenCount(layout, isScreenShared) {
+  function getLayoutByScreenCount(layout , isScreenShared) {
     if (isScreenShared) {
-      return layout;
+        return layout
+    } else {
+        return (getTotal() <= 2) ? layout : 'grid';
     }
-    return (getTotal() <= 2) ? layout : 'grid';
   }
```

Comment:
This is adding back some linting issues that we fixed previously.

Reply:
reverted